newton/tests/test_import_urdf.py

```
@@ -458,6 +461,58 @@ def test_dae_visual_texture_uri_preserved(self):
+        """Verify known pycollada NumPy deprecations do not fail strict warning runs."""
+
+        def make_loader(message: str, module_name: str):
+            namespace = {
+                "DeprecationWarning": DeprecationWarning,
+                "SimpleNamespace": SimpleNamespace,
+                "__name__": module_name,
+                "message": message,
+                "np": np,
+                "warnings": warnings,
+            }
+            exec(
```

Ruff (0.15.17)

[error] Use of `exec` detected

(S102)